version/versions.yaml

```diff
@@ -0,0 +1,4 @@
+- operatorVersion: 1.3.0
+  supportedIstioVersions: 1.3.0
+  recommendedVersions: 1.3.0
```

Comment:
Should be `recommendedIstioVersions` instead of `recommendedVersions` according to yaml tag: https://github.com/istio/operator/blob/cbe894b2178fed887ff8982e2b02e4b0058210e9/pkg/version/version.go#L55

Reply:
Done.